### PR TITLE
feat(accordion): use appropriate aria tags for accessibility

### PR DESCRIPTION
**Pull request by jeremytowne**
The id generation in accordion.js, seems a little sketchy - open to ideas about this.

### CODE REVIEW

**Comment by wesleycho** [comment]
Something similar to https://github.com/angular-ui/bootstrap/blob/3dac5e34504a391ff9e95eb1b0e9772bae9d7448/src/typeahead/typeahead.js#L113 would work I think for generating the id attribute - the Math.random usage is not the greatest solution, but is safe IMO.

**Comment by jeremytowne** [comment]
Ok, I will update it to be similar to that. Should I be adding the aria tags like https://github.com/angular-ui/bootstrap/blob/3dac5e34504a391ff9e95eb1b0e9772bae9d7448/src/typeahead/typeahead.js#L114, or is putting them in the template OK?

**Comment by wesleycho** [comment]
Template is fine

**Comment by jeremytowne** [comment]
It looks like the travis ci build failed because of a timeout issue, is there anyway to run a new build against this?

**Comment by icfantv** [comment]
Looks ok now.

File: src/accordion/accordion.js [hunk 1]
```diff
@@ -91,6 +91,10 @@ angular.module('ui.bootstrap.accordion', ['ui.bootstrap.collapse'])
           }
         }
       };
+
+      var id = 'accordiongroup-' + scope.$id + '-' + Math.floor(Math.random() * 10000);
+      scope.headingId = id + '_tab';
+      scope.panelId = id + '_panel';
```
Comment (wesleycho) [inline src/accordion/accordion.js]: Can you switch these to `-` instead of `_`?

**Comment by wesleycho** [comment]
Just that switch in those two ids to dashes & this should be good.

**Comment by jeremytowne** [comment]
Awsome! Thanks for the feedback.

**Pull request closed without merging**